src/bin/solang.rs

```diff
@@ -241,6 +242,21 @@ fn compile(compile_args: &Compile) {
     if !errors {
         let mut seen_contracts = HashMap::new();
 
+        let authors = if let Some(authors) = &compile_args.package.authors {
+            if target == Target::Solana {
+                eprintln!("warning: the `authors` flag will be ignored for solana target")
+            }
+            authors.clone()
+        } else {
+            vec!["unknown".to_string()]
+        };
+
+        let version = if let Some(version) = &compile_args.package.version {
```

Comment:
Nit, can use `unwrap_or`?

Reply:
It looks better, but it wants to take ownership of `compile_args.package.version`. Then inside `unwrap_or`, it needs to return a reference, so I have to use a let statement beforehand. E.G the final product would look like this: 
`let default_version = "0.0.1".to_string();
 let version = compile_args.package.version.as_ref().unwrap_or(&default_version);`
 So I think `if let some` works better here

Reply:
Not terribly important, the `if let` is fine too